go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/nicksnyder/go-i18n/v2
+module github.com/nicksnyder/go-i18n
```

Comment:
I can’t merge this because the v2 suffix is necessary to avoid breaking backward compatibility with v1. Can you explain the problem you have that motivated this PR?

Reply:
I got some errors when go build my program

Reply:
What are you using to vendor? What version of Go are you using?

Reply:
go version 1.10

Reply:
Can you upgrade to 1.10.3+? I think that might solve your problem.

Reply:
More info here: https://github.com/golang/go/wiki/Modules#how-are-v2-modules-treated-in-a-build-if-modules-support-is-not-enabled-how-does-minimal-module-compatibility-work-in-197-1103-and-111